source/includes/gcp/_routers.md

```diff
@@ -160,7 +160,7 @@ Create a new router
 | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
 | `name`<br/>_string_         | The display name of the router.                                                                                                 |
 | `shortRegion`<br/>_string_  | A short version of the region name.                                                                                             |
-| `shortNetwork`<br/>_string_ | A short version of the network name this router is attached to.                                                        |
+| `shortNetwork`<br/>_string_ | A short version of the network name this router is attached to.                                                                 |
```

Comment:
Would you mind changing this to "A shortened version of the network name that the router is attached to"?

Reply:
Changed in 3573f35ba6eea283f9bc657b837f7e2601e9be49